src/deprecated.jl

```diff
@@ -213,3 +213,7 @@ export HDF5Attribute, HDF5File, HDF5Group, HDF5Dataset, HDF5Datatype,
 ### Changed in PR#691
 # - Make Reference() construct a null reference
 @deprecate_binding HDF5ReferenceObj_NULL HDF5.Reference()
+
+### Changed in PR#694
+export has
+@deprecate_binding has Base.haskey
```

Comment:
```suggestion
@deprecate has(parent::Union{File,Group,Dataset}, path::String) Base.haskey(parent, path)
```
Not completely sure of the semantics, but my understanding is that just deprecating the binding attaches `has` to `Base.haskey` which has a far wider type signature that we shouldn't be duplicating into `has`.

Reply:
good catch